### ISSUE TITLE
The app throws exceptions

### ISSUE THREAD

**Issue by ronyn77** [Maassoft/ColorControl#258]
I hadn't used this app for a long time. When I finally did, I realized I still had version 5.6, or something similar; I can't quite remember. I navigated to NVIDIA Controller -> Current Setting, and when I right-clicked, a menu with various options appeared. I noticed there was an update available, so I decided to update the app to the latest version. During this process, I was prompted to install Microsoft Windows Desktop Runtime 7.0.10 (x64), which I did. Upon its first launch, the app also suggested that I install a certain service, and I complied with this suggestion. However, after completing all these steps, I revisited NVIDIA Controller -> Current Setting, tried right-clicking, and to my surprise, I received an exception, and it no longer functions correctly. I've attempted to download and launch older versions, but unfortunately, they don't work either. What should I do? Please refer to the attached pictures for more details.
![Snag_1302d2](https://github.com/Maassoft/ColorControl/assets/22024600/46e4ce98-66a0-48be-8b82-46ac0ef8999a)
![Snag_132221](https://github.com/Maassoft/ColorControl/assets/22024600/cb5da4ec-205f-4061-9b1c-dad68b5b9aab)

**Comment by Maassoft** [comment]
Thanks for the report. I've noticed this myself as well and had it already fixed in an internal test build.
I've just released v9.6.4.0 which contains the bugfix.

**Comment by ronyn77** [comment]
Yes, now it works. Thank you. I cannot understand another thing. Why, when I press on the Nvidia presets tray icon, it shows them, and they seem to be repeated many times?
![Snag_3a35cb](https://github.com/Maassoft/ColorControl/assets/22024600/ecde7076-f068-4b9e-86f5-5a4ddb3718d5)

**Comment by Maassoft** [comment]
Maybe pressing "Add modes" duplicated the presets (although it shouldn't normally happen). With the new version this button is replaced and you can simply add one new preset.
I think the only way for you is to manually remove all unwanted (duplicated) presets.

**Comment by ronyn77** [comment]
I didn't add all these presets myself, but anyway, I have deleted the excess ones. I tried to play around with the settings and made a couple of adjustments. I have a question: Is there a guide that could help me understand how to configure the various options or at least explain what I should choose and why? For example, there are several options for dithering (spatial dynamic, spatial static, etc.), but I have no idea what the difference is or how to determine which one to use.
